src/components/toast/definition.js

```diff
@@ -28,16 +28,18 @@ let definition = new Definition('toast', Toast, {
     as: OptionsHelper.colors
   },
   propValues: {
-    children: 'test',
+    children: "Talkie's the name, toasting's the game. Anyone like any toast?",
```

Comment:
No toast. Or muffins. We don't like muffins around here. We want no muffins, no toast, no teacakes, no buns, baps, baguettes or bagels, no croissants, no crumpets, no pancakes, no potato cakes and no hot-cross buns and definitely no flapjacks.